board.rb

```diff
@@ -0,0 +1,90 @@
+class Board
+
+    def initialize(player_one, player_two)
+      @player_one = player_one
+      @player_two = player_two
+      if @player_one.get_index == 1
+        @current_player = @player_one
+      else
+        @current_player = @player_two
+      end
+      @winner = nil
+      @is_game_ended = false
+      @positions = [1,2,3,4,5,6,7,8,9]
+    end
+  
+    def positions(positions)
+      @positions = positions
+    end
+    def current_player
+      @current_player
+    end
```

Comment:
You should use [attr_accessors](https://stackoverflow.com/questions/4370960/what-is-attr-accessor-in-ruby)